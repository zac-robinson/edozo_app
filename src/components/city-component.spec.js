import React from 'react';
import { shallow } from 'enzyme';

import City from './city-component';

describe('City component', () => {
  it('should render correctly', () => {
    const component = shallow(<City city='city name' />);

    expect(component).toMatchSnapshot();
  });
});
