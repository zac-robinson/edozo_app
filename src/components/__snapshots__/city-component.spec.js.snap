// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City component should render correctly 1`] = `
<styled.h1
  length={9}
>
  city name
</styled.h1>
`;
